src/component/archive.js

```diff
@@ -0,0 +1,111 @@
+import "../styles.css";
+import { BsPin } from "react-icons/bs";
+import { MdArchive, MdDeleteOutline, MdPushPin } from "react-icons/md";
+import { useNotes } from "../context/noteProvider";
+
+export default function Archive() {
+  const { state, dispatch } = useNotes();
+
+  return (
+    <div className="Label">
+      <div className="label-header">
+        <h4>Archive</h4>
+        <MdDeleteOutline
+          className="p4 text-grey ptr"
+          title="Trash All"
+          onClick={() =>
+            dispatch({
+              type: "ADD_ALL_ARCHIVE_TO_TRASH_NOTES",
+              payload: state.archive
+            })
+          }
+        />
+      </div>
+      {state.archive < 1 ? (
+        <p className="p3 bold center">There is no task in archive.</p>
+      ) : (
+        <ul className="flex-col">
+          {state.archive &&
+            state.archive.map((note) => {
+              return (
+                <li className="card" key={note.id}>
+                  <div className="card__primary-action card__primary-action-column">
+                    {state.pin &&
+                    state.pin.some((item) => note.id === item.id) ? (
+                      <MdPushPin
+                        name="pin item"
+                        title="Pin"
+                        className="badge-up-right-corner p3 text-grey ptr"
```

Comment:
There is inconsistency in class names, above it's joined with '__' and here with '-', if there is no specific reason, try to be consistent in CSS styling